src/nuxeo-cloud/day-to-day/deployment.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Deployment
+review:
+    comment: ''
+    date: '06-19-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 300
+toc: true
+---
+
+## Concept
+
+- **Monitored & Controlled**: The Nuxeo Cloud Operations (NCO) team deploys ALL changes and updates to pre-prod and prod environments
```

Comment:
Replace all of these bullet points with the following text

The Nuxeo Cloud team deploys all changes to Production and Pre-Production Environments via Marketplace Packages. 

JIRA deployment requests should be submitted up to 2 days before the change is required. The request should also include when the deployment should take place. 

Some important notes regarding deployments
- Snapshot or Date-Based Releases are NOT allowed. The Nuxeo Cloud Team only deploys production ready code
- Marketplace Package versions should be used. The Nuxeo Cloud team will NOT deploy the same Marketplace Package to the same environment multiple times

Nuxeo Cloud has Standard Maintenance Windows for all of our Production and Pre-Production Environments. The standard maintenance windows are used to make changes to minimize impact to users. 

The standard 7 days a week US, EU and Japan maintenance windows are identified below.

US: 12AM - 2AM EST
EU: 12AM CET - 2AM CET
Japan 12AM JST - 2AM JST